### PR TITLE
Fix bug: Changed 'VALUE' to 'value' for consistency

### PR DESCRIPTION
**Pull request by RipulHandoo**
## Proposed changes

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

## Special notes for your reviewer:
This fixs [#4176 ](https://github.com/litmuschaos/litmus/issues/4176).
Please review the changes carefully and confirm that the adjustment from 'VALUE' to 'value' is consistent and doesn't introduce any unintended side effects. Thank you for your review!

### CODE REVIEW

**Review by neelanjan00 — APPROVED** [review]

**Review by gdsoumya — CHANGES_REQUESTED** [review]
Can you do a full search and replace for this? It seems like almost all our experiments files in the `mkdocs/docs/` have this, would be good to fix them all in one PR.

**Comment by RipulHandoo** [comment]
> Can you do a full search and replace for this? It seems like almost all our experiments files in the `mkdocs/docs/` have this, would be good to fix them all in one PR.

I search in the [mkdocs/docs](https://github.com/litmuschaos/litmus/blob/master/mkdocs/docs/experiments) and there is only one more [this](https://github.com/litmuschaos/litmus/blob/master/mkdocs/docs/experiments/categories/aws/common/managed-nodegroup.yaml#L27)

I've update the branch PTAL.

**Comment by gdsoumya** [comment]
@RipulHandoo there's surely a lot more 
image

**Comment by RipulHandoo** [comment]
> @RipulHandoo there's surely a lot more image

I'll make changes right away.

**Comment by RipulHandoo** [comment]
@gdsoumya 
I replaced all the occurrences of "VALUE" to "value". 
But there are some other changes as well.
PTAL. if they are alright or not.

File: mkdocs/docs/experiments/categories/aws-ssm/AWS-SSM-experiments-tunables.md [hunk 1]
```diff
@@ -6,7 +6,8 @@ It stressed the `CPU_CORE` cpu cores of the `EC2_INSTANCE_ID` ec2 instance and `
 
 Use the following example to tune this:
 
-[embedmd]:# (https://raw.githubusercontent.com/litmuschaos/litmus/master/mkdocs/docs/experiments/categories/aws-ssm/common/cpu-cores.yaml yaml)
+[embedmd]: # "https://raw.githubusercontent.com/litmuschaos/litmus/master/mkdocs/docs/experiments/categories/aws-ssm/common/cpu-cores.yaml yaml"
```
Comment (gdsoumya) [inline mkdocs/docs/experiments/categories/aws-ssm/AWS-SSM-experiments-tunables.md:9]: Some of these changes will break the docs. It's probably due to your IDE auto formatting, revert the commits and disable your ide formatting first and then run replace. And make sure to only run replace on the env vars and not on any other texts.

**Review by gdsoumya — CHANGES_REQUESTED** [review]

File: mkdocs/docs/experiments/categories/aws-ssm/AWS-SSM-experiments-tunables.md (hunk 1 above)
Comment (RipulHandoo) [reply mkdocs/docs/experiments/categories/aws-ssm/AWS-SSM-experiments-tunables.md:9]: On it.

**Comment by RipulHandoo** [comment]
> LGTM

@gdsoumya 
I am signing off the commits.
But still DCO is failing.
Any suggestions

**Comment by gdsoumya** [comment]
@RipulHandoo try following the steps here https://github.com/litmuschaos/litmus/pull/4253/checks?check_run_id=18290456286

**Comment by RipulHandoo** [comment]
> @RipulHandoo try following the steps here https://github.com/litmuschaos/litmus/pull/4253/checks?check_run_id=18290456286

I tried everything on my end.
Should i close this request and create a new PR for this issue?

**Comment by gdsoumya** [comment]
> I tried everything on my end. Should i close this request and create a new PR for this issue?

No need, have manually set it to pass.

**Review by ispeakc0de — APPROVED** [review]

**Pull request merged**